### PR TITLE
Add API of getting Singleton service's version information

### PR DESCRIPTION
**Pull request by huihuiw01**

### CODE REVIEW

File: g11n-ws/vip-manager-i18n/src/main/resources/application.properties [hunk 1]
```diff
@@ -4,6 +4,10 @@
 ##
 
 #build version information
+build.name=vip-i18n-manager
+build.author=VMware
+build.version=
```
Comment (linr211) [inline g11n-ws/vip-manager-i18n/src/main/resources/application.properties:9]: The value of build.version should be overwritten in the build.gradle and the value should come from the cmd parameter of build creation, right? But i don't see how this work.

Comment (huihuiw01) [reply g11n-ws/vip-manager-i18n/src/main/resources/application.properties:9]: Have synced this with @junw-33455432 , he will overwrite the version value when publish build, as the same with what he did for the existing builddate&buildnumber&branch, so I needn't write such task.

**Review by tigershi — APPROVED** [review]

**Comment by Xiaochao8** [comment]
Why was a new project created for this feature? How about putting this with i18n manager together?

**Comment by huihuiw01** [comment]
> Why was a new project created for this feature? How about putting this with i18n manager together?

This API is simple maybe it's ok put it together with i18n manager, but it's not good practice to put logic into Controller/API layer, this layer is for handle request parameter and response. 
Another more important reason is there maybe other similar API requirement in the feature,  which can be categorized as /about, just like /translation which includes serveral L3 APIs, then they can be added directly under this module. In other word, it's added for extendability.

**Review by linr211 — APPROVED** [review]
The codes are clear to me, fine to ship!

**Pull request merged**